### DISCUSSION TITLE
Why is there no ->capture_default_str() modifier for app.add_flag(...)?

### DISCUSSION THREAD

**Discussion by jpeter42** [CLIUtils/CLI11#938] [Q&A]
I am wondering why there is no `->capture_default_str()` modifier (or similar) for flags. How can I show the default values of flags in the help?

**Comment by phlptp** [comment]
flags are most commonly binary, so in most cases a default value isn't meaningful.  You can of course make a an option that behaves just like a flag using the expected(0,1) modifier.  then it can have a default value and capture_default_str works.
